doc/man7/EVP_SIGNATURE-RSA.pod

```diff
@@ -27,6 +27,8 @@ using EVP_PKEY_sign_init_ex() or EVP_PKEY_verify_init_ex().
 
 =item "digest-check" (B<OSSL_SIGNATURE_PARAM_FIPS_DIGEST_CHECK>) <integer>
 
+=item "verify-digested" (B<OSSL_SIGNATURE_PARAM_FIPS_NO_VERIFY_DIGESTED> <integer>
```

Comment:
these are still not right? And the one ECDSA above also

Reply:
Seems I can't win on the weekend :(